ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/authentication/AuthenticationUtils.java

```diff
@@ -53,6 +53,10 @@ public static boolean isPermitted(
                   foundMatchingPermission.set(true);
                 }
               });
+          // exit if a matching permission was found, no need to keep checking
+          if (foundMatchingPermission.get()) {
+            return foundMatchingPermission.get();
+          }
```

Comment:
Looks like line 67 could be inserted as the else branch here?

Reply:
no because it has to go through all the permissions that the user has